docs/api.rst

```diff
@@ -6,6 +6,30 @@ developers interested in integrating functionality from Hypothesis into their
 own applications.
 
 
+Authorization
+-------------
+
+Some of the API URLs documented below require a valid API token.
+To use these API URLs you should:
+
+1. Generate yourself an API token on your
+   `Hypothesis developer page <https://hypothes.is/profile/developer>`_
+   (you must be signed in to Hypothesis to get to this page).
+
+2. Put the API token in the ``Authorization`` header in your requests to the
+   API.
+
+**Example request**::
+
+   GET /api
+   Host: hypothes.is
+   Accept: application/json
+   Authorization: Bearer 6879-45326b75-b998-431e-a306-70ec121bdf2a
```

Comment:
Could you replace the example tokens with something that looks like the token you're generating (i.e. not a UUID)?